QuestionAnswering/QA_ICL/backend/fastapi_app/controllers/chat.py

```diff
@@ -20,10 +20,15 @@
 
 def binary_search(low: int, high: int, fn: Callable[[int], int], target: int):
     """Performs binary search over the integer space.
+
     Args:
```

Comment:
I notice that for new docstrings you added, you place the short method description below the triple quotation mark, while for this method the short description is on the same line as the triple quotation mark. Please standardise. Also, as per [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#383-functions-and-methods), please decide between using descriptive- or imperative-style for the docstring description.